src/components/HelpModal.vue

```diff
@@ -23,6 +23,25 @@
 				</tr>
 			</thead>
 			<tbody>
+				<tr>
+					<td>{{ t('text', 'New paragraph') }}</td>
+					<td>
+						2x
+						<kbd>{{ t('text', 'Enter') }}</kbd>
+					</td>
+					<td v-if="!isMobile" />
+				</tr>
+				<tr>
+					<td>{{ t('text', 'Hard line break') }}</td>
+					<td>
+						<kbd>{{ t('text', 'Enter') }}</kbd>
+					</td>
+					<td>
```

Comment:
I think this also needs a `v-if="!isMobile"` : 

```suggestion
					<td v-if="!isMobile">
```